packages/next/build/webpack/loaders/next-flight-server-loader.ts

```diff
@@ -7,6 +7,12 @@ function isClientComponent(importSource: string, pageExtensions: string[]) {
   )
 }
 
+function isServerComponent(importSource: string, pageExtensions: string[]) {
```

Comment:
can be merge with isClientComponent since only difference is `client|server` string

Reply:
`isClientComponent` is used in another place already